python/cudf/tests/test_joining.py

```diff
@@ -267,3 +267,32 @@ def test_dataframe_multi_column_join():
         .sort_values(list(pddf_joined.columns))
         .reset_index(drop=True),
         pddf_joined)
+
+
+def test_dataframe_merge_stdout():
```

Comment:
We can remove this test, this slipped through the cracks and will eventually be caught by more rigorous code linting.

Reply:
removed the `test_dataframe_merge_stdout()` as suggested.